### PR TITLE
Refactor emacs_commands.py

### PR DESCRIPTION
**Pull request by rntz**
Move most functionality into `emacs.py`; expose access to the command-name-to-info mapping via talon actions. Simplify emacs_commands.py.

Replace `user.emacs("universal-argument")` with `user.emacs_prefix()`. This now has correct behavior if we need to fall-back to M-x or use RPC.

Allow `emacs_commands.csv` to specify the keybindings used for `user.emacs_prefix()` and for `execute-extended-command` (used by `user.emacs()` if no keybinding is found). This reduces the number of places a user needs to customize to use nonstandard keybindings.

### CODE REVIEW

File: apps/emacs/emacs.talon [hunk 1]
```diff
@@ -14,7 +14,7 @@ execute <user.text>$:
     user.emacs_meta_x()
```
Comment (nriley) [inline apps/emacs/emacs.talon:14]: There's still a few uses of `emacs_meta_x` in emacs.talon that need updating.

**Review by nriley — CHANGES_REQUESTED** [review]
Looks good other than one nit!

File: apps/emacs/emacs.talon (hunk 1 above)
Comment (rntz) [reply apps/emacs/emacs.talon]: Ok, should have fixed this now.

**Comment by rntz** [comment]
I don't understand why the CI tests are failing. @phillco any idea what's up?

**Comment by rntz** [comment]
I don't understand why the CI tests are failing. @phillco any idea what's up?

**Review by nriley — APPROVED** [review]

**Pull request merged**